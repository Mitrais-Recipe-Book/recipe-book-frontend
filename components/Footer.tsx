export default function Footer() {
  return (
    <footer className="footer">
      <div className="container">
        <div className="content text-center">
          <strong>Copyright Â©2022 Recipy</strong>
        </div>
      </div>
    </footer>
  );
}
