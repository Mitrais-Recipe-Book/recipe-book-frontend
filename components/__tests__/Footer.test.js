import Footer from "../Footer";
import { render } from "@testing-library/react";
import renderer from "react-test-renderer";

export {};

describe("Test Footer", () => {
  it("render without crashing", () => {
    render(<Footer />);
  });
  it.todo("has recipy copyright");
  it("mathc the snapshot", () => {
    const tree = renderer.create(<Footer />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
