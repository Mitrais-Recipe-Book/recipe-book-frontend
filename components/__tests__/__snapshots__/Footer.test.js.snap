// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test Footer mathc the snapshot 1`] = `
<footer
  className="footer"
>
  <div
    className="container py-10 mx-auto text-center"
  >
    <strong>
      Copyright Â©2022 Recipy
    </strong>
  </div>
</footer>
`;
